test/functional/android/helper/test_helper.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+SLEEPY_TIME = 3
+
+
+def wait_for_element(driver, locator, value, timeout=SLEEPY_TIME):
+    '''
+    Wait until the element located
+
+    :param driver: Webdriver
+    :param locator: Locator like WebDriver, Mobile JSON Wire Protocol (e.g. MobileBy.ACCESSIBILITY_ID)
+    :param value: Query value to locator
+    :param timeout: Maximum time to wait the element. If time is over, TimeoutException is thrown
+    :return: None
+    '''
+    WebDriverWait(driver, timeout).until(
```

Comment:
Does `presence_of_element_located` returns an element?
If so, what about returning the result of `WebDriverWait` as this method's returns value?

Reply:
Seems good, will do.